src/wrapper-types.jl

```diff
@@ -122,10 +122,10 @@ function wrap_func_expr(mod, expr, wrap_arrays = true)
                     (elT) -> :(AbstractArray{T} where {T <: $elT})
                 end
                 if has_symwrapper(eT)
-                    Ts = (Ts..., # _arr_type_fn(:(Symbolics.SymbolicUtils.Symbolic{<:$eT})), 
+                    Ts = (Ts..., _arr_type_fn(:(Symbolics.SymbolicUtils.Symbolic{<:$eT})), 
                     _arr_type_fn(wrapper_type(eT)))
-                # else
-                #     Ts = (Ts..., _arr_type_fn(:(Symbolics.SymbolicUtils.Symbolic{<:$eT})))
+                else
+                    Ts = (Ts..., _arr_type_fn(:(Symbolics.SymbolicUtils.Symbolic{<:$eT})))
```

Comment:
why was it commented?

Reply:
I'm pretty sure I disabled it temporarily in #1197 because of the issues with this (and the `AbstractArray{Num}`) dispatch for registered `Base` functions, but forgot to uncomment it once it was resolved.